File: lib/src/main/java/de/tum/in/tumcampusapp/models/IdentitySet.java
package de.tum.in.tumcampusapp.models;

import org.simpleframework.xml.ElementList;
import org.simpleframework.xml.Root;

import java.util.List;

@Root(name = "rowset")
public class IdentitySet {

	@ElementList(inline = true)
	private List<Identity> ids;

	@Override
	public String toString() {
		if(ids==null || ids.size()==0)
		     return null;
		return ids.get(0).toString();
	}
}

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/person/model/IdentitySet.kt
package de.tum.`in`.tumcampusapp.component.tumui.person.model

import com.tickaroo.tikxml.annotation.Element
import com.tickaroo.tikxml.annotation.Xml

@Xml(name = "rowset")
data class IdentitySet(@Element val ids: List<Identity>) {

    override fun toString(): String =
            if (ids.isEmpty())
                "null"
            else ids[0].toString()

}
